### PR TITLE
Education benefits JSON schema

### PR DESCRIPTION
**Pull request by jbalboni**
Here's the WIP education benefits form json schema.

### CODE REVIEW

**Comment by lihanli** [comment]
can we have ssn be just a string with 9 digits? otherwise later when we are repopulating the form or something we have to check which format it's in and do 2 different things

**Comment by lihanli** [comment]
linter was complaining about 
```
const states = _.uniq(_.flatten(_.values(options.states)).map(object => object.value));
```
not being used so i commented it out

**Comment by lihanli** [comment]
hey, people are complaining about the tests taking too long, let's close and then reopen when we are ready to merge

File: src/js/edu-benefits/schema/edu-benefits.js [hunk 1]
```diff
@@ -0,0 +1,397 @@
+// Relocate options-for-select.
+const options = require('../../hca/utils/options-for-select');
+const _ = require('lodash');
+
+// const states = _.uniq(_.flatten(_.values(options.states)).map(object => object.value));
```
Comment (lihanli) [inline src/js/edu-benefits/schema/edu-benefits.js]: @jbalboni do you know if this is needed? linter caught it as an unused variable

Comment (jbalboni) [inline src/js/edu-benefits/schema/edu-benefits.js]: It looks like they changed the way they're flattening the states list, so I don't think so.

**Comment by lihanli** [comment]
closing because tests are backed up, i think i'm done though

**Comment by lihanli** [comment]
@jbalboni ok i think i'm done

File: src/js/edu-benefits/schema/edu-benefits.js [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+      type: 'string'
+    },
+    dateRange: {
+      type: 'object',
+      properties: {
+        from: {
+          $ref: '#/definitions/date'
+        },
+        to: {
+          $ref: '#/definitions/date'
```
Comment (jbalboni) [inline src/js/edu-benefits/schema/edu-benefits.js]: I wonder if we need to handle "present" (like 5/5/2011 - present).

File: src/js/edu-benefits/schema/edu-benefits.js [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+      type: 'object',
+      properties: {
+        accountType: {
+          type: 'string',
+          'enum': ['checking', 'savings']
+        },
+        bankName: {
+          type: 'string'
+        },
+        routingNumber: {
```
Comment (jbalboni) [inline src/js/edu-benefits/schema/edu-benefits.js]: Routing numbers are 9 digits, so we could limit that here.

File: src/js/edu-benefits/schema/edu-benefits.js [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+          type: 'string'
+        },
+        accountNumber: {
+          type: 'string'
+        }
+      }
+    },
+    previouslyFiledClaimWithVa: {
```
Comment (jbalboni) [inline src/js/edu-benefits/schema/edu-benefits.js]: Looking at VONAPP, this question expands into a bunch of checkboxes asking what type of claim and the file number:
screen shot 2016-08-22 at 4 11 03 pm

**Comment by jbalboni** [comment]
@lihanli Looks good to me. I made a few comments, but if there are changes for those we could just make them in the new repo.

**Pull request closed without merging**